app/redux/actions/updateStatusAction.ts

```diff
@@ -1,26 +1,23 @@
-import { Account, BalanceStatus, SelectedAccount, WalletStatus } from '../../types';
+import { Account, BalanceStatus, SelectedAccount } from '../../types';
 
 export const UPDATE_WALLET_STATUS = 'UPDATE_WALLET_STATUS';
 
 export type UpdateStatusAction = {
   payload: {
     selectedAccount: SelectedAccount;
-    walletStatus: WalletStatus;
   };
   type: 'UPDATE_WALLET_STATUS';
 };
 
 export const updateStatusAction = (
   account: Account,
-  balanceStatus: BalanceStatus,
-  walletStatus: WalletStatus
+  balanceStatus: BalanceStatus
```

Comment:
Optional/Equivalent: Keep a trailing comma so future changes to the params do not require modifying this line
```suggestion
  balanceStatus: BalanceStatus,
```